dotcom-rendering/src/web/components/Metrics.importable.tsx

```diff
@@ -0,0 +1,115 @@
+import type { ABTest, ABTestAPI } from '@guardian/ab-core';
+import {
+	bypassCommercialMetricsSampling,
+	initCommercialMetrics,
+} from '@guardian/commercial-core';
+import {
+	bypassCoreWebVitalsSampling,
+	initCoreWebVitals,
+} from '@guardian/core-web-vitals';
+import { getCookie } from '@guardian/libs';
+import { dcrJavascriptBundle } from '../../../scripts/webpack/bundles';
+import type { ServerSideTestNames } from '../../types/config';
+import { integrateIma } from '../experiments/tests/integrate-ima';
+import { useAB } from '../lib/useAB';
+import { useAdBlockInUse } from '../lib/useAdBlockInUse';
+import { useOnce } from '../lib/useOnce';
+
+type Props = {
+	enabled: boolean;
```

Comment:
Since `enabled` only controls whether CommercialMetrics are recorded, could the name be changed to reflect this? It may be confusing to pass `false` through here and find that CoreWebVitals are still recorded.

Reply:
Whoops, duplicate. Definitely need to refresh the page before making a comment 😅